### PR TITLE
Add join program test

### PR DESCRIPTION
**Pull request by d8660091**

### CODE REVIEW

File: tests/programs/programs.js [hunk 1]
```diff
@@ -63,6 +65,85 @@ module.exports = {
...
+      .click('xpath', `//div[contains(text(), '${TEST_USERS.PROGRAM_ADMIN_SINGLE.email}')][1]`)
+      .frame('preview-html', function() {
+        this.getAttribute('xpath', "//a[contains(text(), 'JOIN THE PROGRAM')]", 'href', r => {
+          inviteId = r.value.match(/[^\/]*$/)[0];
+          this.url(buildUrl(`/submission/program/join/login/${inviteId}`))
+            .waitForElementVisible('#google-login', 10000)
+            .click('#google-login')
+            .deleteCookies()
+            .refresh()
+            .waitForElementVisible('input[type="email"]')
```
Comment (hlminh2000) [inline tests/programs/programs.js]: are we navigating the google login UI here? I haven't tried that out before myself but just curious cause I thought that didn't work

Comment (d8660091) [reply tests/programs/programs.js]: Yes, it's navigating to google login UI. The input does not appear when the test browser has Google cookie, but after deleting cookies and refreshing, the input would appear.

Comment (hlminh2000) [reply tests/programs/programs.js]: gotcha! @joneubank would that work with our login test as well? could give it a shot if we haven't tried :)

Comment (joneubank) [reply tests/programs/programs.js]: If this hack works then that is huge... I'll run a trial of this script on browserstack to see if it gets through the issue

**Pull request merged**